src/main/java/net/openhft/chronicle/bytes/Bytes.java

```diff
@@ -998,14 +998,13 @@ default boolean isEqual(@Nullable String other)
      * <p>
      * This operation is useful to free up space for writing by discarding bytes that have already
      * been read.
-     * 
+     * <p>
+     * If the underlying Bytes have been released, this does nothing.
      *
      * @return This Bytes object, for method chaining.
-     * @throws IllegalStateException if this Bytes object has been previously released.
```

Comment:
why change this?

Reply:
This was changed to reflect the actual implementation.
However, the implementation should fail fast and this has been covered by another change.